gap/SkeletalFinSetsForCAP.gi

```diff
@@ -281,7 +281,7 @@ end );
 
 ##
 AddMorphismDatum( SkeletalFinSets,
-  function ( SkeletalFinSets, map )
+  function ( cat, map )
```

Comment:
Please also change this in `ObjectConstructor` and `MorphismConstructor` so things are consistent.